ampersand-filter-tracker.js

```diff
@@ -119,10 +152,20 @@
 
       filterContainers.select('span.ampersand-filter-tracker-item-text')
         .text(function(d) { return d.output.call(d.model); });
+
+      this.div
+        .style('display', _.any(handles, function(handle) { return handle.active; }) ? undefined : 'none');
```

Comment:
you can replace the function in any with `'active'`, like this: `_.any(handles, 'active')`